### ISSUE TITLE
Pick/OnionPeel yield different parallel/serial results on data resulting from automatic domain decomposition

### ISSUE THREAD

**Issue by aowen87** [aowen87/FOO#276]
For formats that do automatic domain decomposition (like CHGCAR, subset of VASP and H5Part).
Picking or performing OnionPeel yields different results in parallel than in serial.

If this is perfectly acceptable and/or the expected behavior, then this ticket can be closed.

If not, then some infrastructure may need to be introduced that will allow Pick/OnionPeel to work from a "global" id set
that is provided by the formats.

Other queries may also be impacted, I haven't tested.

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Kathleen Biagas
Original creation: 02/24/2011 03:24 pm
Original update: 03/02/2011 07:13 pm
Ticket number: 633
